src/attached_probe.cpp

```diff
@@ -515,8 +501,20 @@ bool AttachedProbe::resolve_offset_uprobe(bool safe_mode, bool has_multiple_aps)
   if (func_offset == 0)
     return true;
 
-  check_alignment(
-      probe_.path, symbol, sym_offset, func_offset, safe_mode, probe_.type);
```

Comment:
What about safe_mode? 

I think this PR takes away ability for `--unsafe` to force an attachment. Is that the intention?

Reply:
Good call, then :)

Maybe point it out in commit msg in case someone wonders down the line where it went. Changelog entry about that would be good too